src/context.ml

```diff
@@ -540,11 +540,42 @@ let create ~(kind : Kind.t) ~path ~env ~env_nodes ~name ~merlin ~targets
   in
   native :: List.filter_opt others
 
+let extend_paths t ~env =
+  let module Eval =
+    Ordered_set_lang.Make(String)
+      (struct
+        type t = string
+        type key = string
+        let key x = x
+      end)
+  in
+  let t =
+    let f (var, t) =
+      let parse ~loc:_ s = s in
+      let standard = Env.path env |> List.map ~f:Path.to_string in
+      var, Eval.eval t ~parse ~standard
+    in
+    List.map ~f t
+  in
+  let vars =
+    let cwd = Sys.getcwd () in
+    let of_filename_relative_to_cwd s =
```

Comment:
What about: `Path.of_string s |> Path.to_absolute_filename`? That would make the path be interpreted relative to the workspace root, which is consistent with how other paths are interpreted.

Reply:
I thought the current code did  that (isn't `cwd` the workspace root?), but anyway your suggestion is much cleaner.

Reply:
Yep, the current does that but it seems better to me to leave that job to `Path`

Reply:
Fixed.